Sources/SwiftCompilerPluginMessageHandling/PluginMessages.swift

```diff
@@ -23,7 +23,8 @@ public enum HostToPluginMessage: Codable {
     macro: PluginMessage.MacroReference,
     macroRole: PluginMessage.MacroRole? = nil,
     discriminator: String,
-    syntax: PluginMessage.Syntax
+    syntax: PluginMessage.Syntax,
+    lexicalContext: [PluginMessage.Syntax]?
```

Comment:
This is an API-breaking change. We should at least release-note it.

Reply:
I'll default and release-note it.